react/ContextHeader/styles.styl

```diff
@@ -0,0 +1,31 @@
+@require 'settings/palette.styl'
+
+.context-header
+  margin 1rem 2rem
```

Comment:
There's a mixin now, to simplify your math. you can just write `margin rem(16 32)`, the unit is implicitely pixel.